### ISSUE TITLE
getImageInfo returns null altough file exists in File namespace

### ISSUE THREAD

**Issue by syfantid** [fastily/jwiki#1]
I'm using getImageInfo with the String parameter: "ShadowHoursDVDCoverAmazon.jpg" (I also tried "File:ShadowHoursDVDCoverAmazon.jpg" with no luck).
It exists in the File namespace (see here: https://en.wikipedia.org/wiki/File:ShadowHoursDVDCoverAmazon.jpg). However it returns a null object instead of ImageInfo object. Is it a bug maybe?

**Comment by fastily** [comment]
Hi syfantid, and thanks for your report!  I have not been able reproduce the output you are describing, but here's the code snippet I've been testing this with:

```java
package enwp;

import jwiki.core.Wiki;
import jwikix.util.WikiGen;

public class TestImageInfo
{
	public static void main(String[]args)
	{
		Wiki wiki = WikiGen.wg.get("FastilyClone", "en.wikipedia.org");
		System.out.println(wiki.getImageInfo("File:ShadowHoursDVDCoverAmazon.jpg"));
	}
}
```
It prints:
`[[ user : Tchernomush, title : File:ShadowHoursDVDCoverAmazon.jpg, summary : Uploading a piece of non-free cover art using [[Wikipedia:File_Upload_Wizard|File Upload Wizard]], timestamp : 2013-06-07T20:48:33Z ]]`

Also, by any chance are you using the 1.1 release?

**Comment by syfantid** [comment]
Yes, I've been using the 1.1 release, where I can't find WikiGen, so I've been using just Wiki to initialize the connection. Should I compile the whole project after the latest commit?

**Comment by fastily** [comment]
Apologies, I should have clarified that.  WikiGen is from [jwiki-extras](https://github.com/fastily/jwiki-extras), and that particular call is the equivalent of

```java
Wiki wiki = new Wiki("FastilyClone", "Password", "en.wikipedia.org");
```

I have posted a v1.2 [release](https://github.com/fastily/jwiki/releases), as I have made some changes to `ImageInfo` since v1.1.  This is a minor breaking change, since `Wiki.getImageInfo()` now returns a `List<ImageInfo>` instead of just a single `ImageInfo` object.  Let me know if this fixes the problem for you :)
